Trinap/Sources/Utils/UserDefaultKey.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  UserDefaultKey.swift
+//  Trinap
+//
+//  Created by Doyun Park on 2022/12/07.
+//  Copyright © 2022 Trinap. All rights reserved.
+//
+
+import Foundation
+
+struct UserDefaultKey {
+    static let isPhotographer = "isPhotographer"
+}
```

Comment:
static으로 Key같은 것 모아놓을 때 `enum` 사용하면 좋을 것 같습니다!

struct나 class로 선언하게 되면 `UserDefaultKey` 인스턴스 자체를 생성할 위험(?)이 있는데,
`enum`으로 선언하면 인스턴스를 아예 생성하지 못하게 되어서 좋다고 들었습니다..!

```suggestion
enum UserDefaultKey {
    static let isPhotographer = "isPhotographer"
}
```